src/components/TimeSelectInput/index.d.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentType } from 'react';
+import { BaseProps } from '../types';
+
+export interface TimeSelectInputProps extends BaseProps {
+    value?: string;
+    onChange?: (time: string) => void;
+    label?: ReactNode;
+    hideLabel?: boolean;
+    required?: boolean;
+    bottomHelpText?: ReactNode;
+    error?: ReactNode;
+    readOnly?: boolean;
+    disabled?: boolean;
+    tabIndex?: number | string;
+    onClick?: (event: MouseEvent<HTMLInputElement>) => void;
+    onFocus?: (value: string) => void;
+    onBlur?: (value: string) => void;
```

Comment:
There are inconsistencies here, currently, blur sends a string and it is fired only once like if it was only one input (good until here), but initially blur without any value it sends `null` instead of an empty string.
In the case of the focus, it behaves completely different, it fires an event object instead of the string and it fires every time for each input.
Let me check but I think we should send on both a string empty when no value or incomplete or the value when correct, and treat the input as only one, it means it should not send focus for each input